cider/exec/nextgen/operators/ArrowSourceNode.cpp

```diff
@@ -41,7 +41,7 @@ void ArrowSourceTranslator::codegen(Context& context) {
   JITFunction* func = context.query_func_;
   auto inputs = node_.getOutputExprs();
   // get ArrowArray pointer
-  auto arrow_pointer = func->getArgument(0);
+  auto arrow_pointer = func->getArgument(1);
```

Comment:
Minor hint: Magic number. Let's point out where func signature is defined.

Reply:
Get it, fix